met-web/src/App.tsx

```diff
@@ -25,6 +26,8 @@ const App = () => {
         UserService.initKeycloak(dispatch);
     }, [dispatch]);
 
+    localStorage.setItem('apiurl', String(AppConfig.apiUrl));
```

Comment:
hm..General principle is to stick to sessionstorage unless there is a motive to store items in localstorage ;like track a user even if he closes browser..I guess here good enough to be in sessionstorage rite?

Reply:
Done, replaced by sessionStorage